### PR TITLE
Add itolparser

### PR DESCRIPTION
**Pull request by boasvdp**
Describe your pull request here

This pull request adds itolparser, a tool that writes annotation files for iTOL (https://itol.embl.de/) from a user-supplied table. The conda package was created from the pypi package.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by boasvdp** [comment]
@BiocondaBot please add label

File: recipes/itolparser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: b4271b168e24859ea806388e934c6bb1eda18ee70ab6c8ae236dbb2ca17c9ebf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - colorbrewer
```
Comment (simonbray) [inline recipes/itolparser/meta.yaml:19]: It should work fine with only python and pip under host

File: recipes/itolparser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - python >=3.6
+  run:
+    - colorbrewer
+    - numpy
+    - pandas
+    - python
+
+test:
+  imports:
+    - bin
```
Comment (simonbray) [inline recipes/itolparser/meta.yaml:32]: Not really related to the conda recipe, but itolparser would be a better name for this import, could you rename the directory in your repo?

You can also create an entry point for your itolparser script: https://packaging.python.org/guides/distributing-packages-using-setuptools/#entry-points

File: recipes/itolparser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+about:
+  home: "https://github.com/boasvdp/itolparser"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (simonbray) [inline recipes/itolparser/meta.yaml:38]:
```suggestion
  license_file: LICENSE
```

File: recipes/itolparser/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+test:
+  imports:
+    - bin
+
+about:
+  home: "https://github.com/boasvdp/itolparser"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "A"
```
Comment (simonbray) [inline recipes/itolparser/meta.yaml:39]:
```suggestion
  summary: "Small script to produce iTOL colorstrip metadata files from a table"
```

File: recipes/itolparser/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  doc_url: 
+  dev_url:
```
Comment (simonbray) [inline recipes/itolparser/meta.yaml:41]:
```suggestion

```

**Comment by boasvdp** [comment]
Thanks @simonbray, I'm learning a lot from your comments (first time Bioconda contributor). I have included all your suggestions in the new version. I have also updated the README and added a version flag.

**Comment by simonbray** [comment]
@bioconda-bot please fetch artifacts

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@bioconda-bot please merge

Thanks @boasvdp, this looks good to me.

**Pull request merged**